templates/package.json.hbs

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@tatumio/tatum-{{slug}}",
+  "version": "2.0.0-alpha.83",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "test": "jest",
+    "clean": "rimraf dist* tsconfig.tsbuildinfo",
+    "build": "tsc",
+    "build:watch": "tsc -w",
+    "prepublish": "npm run clean && npm run build",
+    "lint": "eslint ./src --ext .ts --fix"
+  },
+  "dependencies": {
+    "@tatumio/tatum-core": "file:../tatum-core",
+    "@tatumio/tatum-defi": "file:../tatum-defi",
+    "@tatumio/tatum-ledger": "file:../tatum-ledger",
```

Comment:
Submodules are not depending on tatum-defi and tatum-ledger anymore, so these two lines should be removed.